airbyte-config/config-models/src/main/resources/types/WellKnownTypes.yaml

```
@@ -0,0 +1,51 @@
+# Regexes are purely illustrative and need to be workshopped: BC dates shouldn't require 4-digit years; years may have >=5 digits; etc
+definitions:
+  String:
+    type: string
+    description: Arbitrary text
+  BinaryData:
+    type: string
+    description: >
+      Arbitrary binary data. Represented as base64-encoded strings in the JSON transport.
+      In the future, if we support other transports, may be encoded differently.
+    # All credit to https://stackoverflow.com/a/475217 for this pattern
+    pattern: (?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?
+  Date:
+    type: string
+    # format: date is a superset of what we want, so we cannot use it here (e.g. it accepts 2-digit years)
+    pattern: \d{4}-\d{2}-\d{2}( BC)?
```

Maybe add an example for legibility: 

```suggestion
    pattern: \d{4}-\d{2}-\d{2}( BC)? # e.g. 2022-10-03
```

---

The same goes for the other date/time fields below